functions.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+function legoCollection (array $legoSets) : string {
+
+    if($legoSets === []){
+        $displayLego = 'There is an error with the database, please try again later.';
+    } else {
```

Comment:
to prevent the rest of the code being inside an else statement just have the if statement above work in isolation with a return in it.